gpflow/base.py

```diff
@@ -192,55 +202,55 @@ def assign(
         )
 
     @property
-    def is_tensor_like(self):
+    def is_tensor_like(self) -> UNKNOWN:
```

Comment:
`UNKNOWN` not `bool`?

Reply:
i don't know what the actual API asks for. I did it this way to avoid working out types from what was in this file, rather work them out from the tf docs

if this method doesn't normally return bool, it's a misnomer, but I wanted to be sure

Reply:
changed to bool